.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,11 @@
 blank_issues_enabled: true
 contact_links:
+- name: Report a bug
```

Comment:
@jandubois 
Renamed the `name` tag from `config.yml` in order to be different from `bug_report.yml`, but I have a feeling GitHub will display both entries on Issues tab:
* A button from `./ISSUE_TEMPLATE/bug_report.yml`
* and a button from the `config.yml`